data/frame_json.gen.go

```diff
@@ -3,10 +3,11 @@ package data
 import (
 	"github.com/apache/arrow/go/v13/arrow"
 	"github.com/apache/arrow/go/v13/arrow/array"
-	jsoniter "github.com/json-iterator/go"
+
+	sdkjsoniter "github.com/grafana/grafana-plugin-sdk-go/data/utils/jsoniter"
```

Comment:
How can I regenerate this file?

Reply:
I think it's executing frame_json_test.go, did you solve this?

Reply:
`frame_json.gen.go` is being used in `frame_json.go`. Because of its name, I thought it was generated. Tests are passing though.